spark-wrapper/spark-3.0/pom.xml

```diff
@@ -9,7 +9,7 @@
         <relativePath>../../pom.xml</relativePath>
     </parent>
 
-    <artifactId>spark-wrapper-spark-3.0</artifactId>
+    <artifactId>spark-wrapper-spark-3.0_${spark.version.release}</artifactId>
```

Comment:
Is here ${scala.version}?
Because you said "spark-wrapper-spark-3.x will change to spark-wrapper-spark-3.x_${scala_version}".

Reply:
+1

Reply:
typo,sorry